DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -4246,7 +4258,6 @@
 				859872221F5743AF00041CB8 /* FireAnimation */,
 				980891A122369ADB00313A70 /* FeedbackUserText.swift */,
 				985AAE4424899369007A43EC /* HomeScreenTransition.swift */,
-				1EF24234273BB9D200DE3D02 /* IntervalSlider.swift */,
```

Comment:
Is this a deliberate removal? I don't see it in the file diff

Reply:
Good catch! I think something went awry with a merge commit from develop along with flattening all the PRs. Fixed